apps/frontend/src/app/PageTeams/index.tsx

```diff
@@ -20,17 +20,17 @@ import {
 import { Suspense, useEffect, useMemo, useState } from 'react'
 import { useNavigate } from 'react-router-dom'
 import TeamCard from './TeamCard'
+import SortByButton from '../Components/SortByButton'
+import { teamSortConfigs, teamSortMap } from './TeamSort'
+import { teamSortKeys } from '@genshin-optimizer/gi/db'
+import { sortFunction } from '@genshin-optimizer/common/util'
 const columns = { xs: 1, sm: 2, md: 3, lg: 4, xl: 4 }
 
 // TODO: Translation
 
 export default function PageTeams() {
   const database = useDatabase()
   const [dbDirty, forceUpdate] = useForceUpdate()
-  const teamIds = useMemo(
-    () => dbDirty && database.teams.keys,
-    [dbDirty, database]
-  )
   const navigate = useNavigate()
   // Set follow, should run only once
   useEffect(() => {
```

Comment:
in the `followAny`, you should also add `(r === 'new' || r === 'remove' || r === 'update')` to allow triggering of dbDirty from updating entries